business/team.js

```diff
@@ -289,6 +289,40 @@ class Team {
       common.createInstancesCallback(this, this.memberFromEntity, callback));
   }
 
+  checkRepositoryPermission(repositoryName, options, callback) {
+    if (!callback && typeof (options) === 'function') {
```

Comment:
Thanks. In this codebase we do use parens for `typeof`. Agreed I do not think it's required.